addons/l10n_pl/data/l10n_pl_chart_data.xml

```diff
@@ -5,11 +5,11 @@
 
         <record id="pl_chart_template" model="account.chart.template">
             <field name="name">Polska - Plan kont</field>
-            <field name="code_digits">6</field>
+            <field name="code_digits">8</field>
```

Comment:
Gné? You are reducing the length of the prefix 11.000.000 => 11.000.00 but increasing code_digits 6 => 8.. it is a mistake?

Reply:
No, it is not by mistake. 
I put a length of 8 as it is the length of the accounts in our CoA.
For the prefix, we need a length of 7, as the accounts automatically generated will have an 8th digit
=> the Bank Suspense Account generated will be 11.000.001, so a length of 8
If we keep the prefix as 11.000.000, it would have generated 11.000.0001 which would be of length 9